### PR TITLE
[Sign and submit] Update Beta design and content

### LINKED ISSUE #476: [Page] Sign & submit - Phase 2 - latest design/content
- [ ] Task: Improve initial loading state
- [ ] Task: Integrate latest designs
- [ ] Task: Update content
- [x] Fix: Render all 4 address lines
- [x] Fix: State not rendering
- [ ] Fix: Update POC form to send State sentence-cased (or abbreviated?)
- [ ] Feat: Integrate total SBLAR lines from API (separate ticket)

### PR DESCRIPTION
**Pull request by meissadia**
Close #476

## Changes

- Disables all checkboxes
- Disables `Submit` button
- Removes `Clear form` button
- Adds top/bottom alerts
- Updates section headings and descriptions

## Notes
- Bug: Address lines 3 and 4
  - addressed in a separate PR
- Bug: State name not displayed
  - addressed in a separate PR

## Screenshots - Updated May 14, 12:35am ET
screencapture-localhost-8899-filing-2024-123456TESTBANKSUB456-submit-2024-05-14-10_37_11

### CODE REVIEW

**Comment by natalia-fitzgerald** [comment]
@meissadia 
It's awesome to see this taking shape! Can you sense my enthusiasm?!!

I did a quick review and noticed a few details to fix:
- In intro text add underline to reg link 
- In form-level warning alert change heading to “You have reached the final step of the beta filing process”  
- Confirm that “upload a new file” link is an inline link and not a list link  
- Change section heading from “Confirm financial institution details” to “Confirm your financial institution details”  
- Update success box to the version that’s in Figma
- Update the sign and certify box to match what's in Figma

For the disabled checkboxes does the color of the text change as a part of disabling? I can see what the Figma component does but I'm curious what's in the code. 

For the disabled checkboxes I had an error in my file so I wasn't seeing the text at 670px width. What this means is that there are a few words on their own line that I didn't intend for. I have flagged these for Angel and we will correct them right away. I did go ahead and correct my Figma file to reflect the correct width. My bad. 
Screenshot 2024-05-10 at 4 42 09 PM
Screenshot 2024-05-10 at 4 42 11 PM
Screenshot 2024-05-10 at 4 43 11 PM

**Comment by meissadia** [comment]
@natalia-fitzgerald Screenshot updated with latest changes
- [x] In intro text add underline to reg link 
- [x] In form-level warning alert change heading to “You have reached the final step of the beta filing process”  
- [x] Confirm that “upload a new file” link is an inline link and not a list link  
  - For functionality these links are buttons displayed as links, so I've updated the font-weight to match inline links
- [x] Change section heading from “Confirm financial institution details” to “Confirm your financial institution details”  
- [x] Update success box to the version that’s in Figma
- [x] Update the sign and certify box to match what's in Figma

File: src/components/CommonLinks.tsx [hunk 1]
```diff
@@ -12,25 +13,86 @@ function NIC(): ReactElement {
...
+  const { lei, year } = useParams();
+  const navigate = useNavigate();
+
+  const onClick = (): void => navigate(`/filing/${year}/${lei}/upload`);
+  return (
+    <Button className={className} asLink onClick={onClick} label={label} />
+  );
+}
+
+const FigSectionUrls: Record<string, string> = {
```
Comment (billhimmelsbach) [inline src/components/CommonLinks.tsx:77]: Thanks you for this lol! I look forward not having to type these in. 😅

File: src/pages/Filing/FilingApp/FilingSubmit.tsx [hunk 2]
```diff
@@ -116,8 +121,30 @@ export function FilingSubmit(): JSX.Element {
...
+                <p>
+                  An authorized representative of your financial institution
+                  with knowledge of the data must certify the accuracy and
+                  completeness of the data reported pursuant to{' '}
+                  <Links.FIG section='§ 1002.109(a)(1)(ii)' />.
+                </p>
+              }
             />
+            <Alert
+              status='warning'
```
Comment (billhimmelsbach) [inline src/pages/Filing/FilingApp/FilingSubmit.tsx:134]: Can we get an `aria-live=polite` [on this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) alert box? I don't want to yell at them about it, but it's a nice thing to do before they try to skip into a disabled form.

**Review by billhimmelsbach — COMMENTED** [review]
Nice work implementing all the content changes here. Small comments, then ready to go!

**Review by billhimmelsbach — APPROVED** [review]
Tested that aria-live change with a screen reader and it's working! 👍

**Pull request merged**